testing/integration/integration_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"context"
+	"testing"
+)
+
+func TestIntegrationUtil(t *testing.T) {
```

Comment:
What does this do?

Reply:
It demonstrates that the integration_util works.  Right now it just starts everything.  This will eventually be a smoke test or something similar.

Reply:
Ah got it. Can you drop a comment in there nothing it's a wi

Reply:
Done.